README.md

```diff
@@ -176,6 +176,12 @@ See the [Epsilon Tutorial](https://symforce.org/notebooks/epsilon_tutorial.html)
 
 We will model this problem as a factor graph and solve it with nonlinear least-squares.
 
+For this example in particular, we need to make sure we're on the `symengine` symbolic backend, and not `sympy`:
+
+```python
+assert symforce.get_backend() == "symengine"
+```
```

Comment:
we should be able to kill this after #163 ? probably just worth doing now

Reply:
Killed, that one should merge before or same time as this one